libr/bin/bfile.c

```diff
@@ -1,6 +1,7 @@
 /* radare2 - LGPL - Copyright 2009-2018 - pancake, nibble, dso */
 
 #include <r_bin.h>
+#include <bin_private.h>
```

Comment:
Maybe we should have an i/ directory to put the private/internal includes. The bin_ prefiz is implicit so we can use a different name for this

Reply:
Do you mean in `libr/bin/i`?

Reply:
yes